pkg/common/configs/config.go

```diff
@@ -227,5 +193,14 @@ func GetConfigurationString(requestBytes []byte) string {
 	return strings.ReplaceAll(conf, checksum, "")
 }
 
-// Default loader, can be updated by tests
-var SchedulerConfigLoader LoadSchedulerConfigFunc = loadSchedulerConfigFromFile
+// DefaultSchedulerConfig contains the default scheduler configuration; used if no other is provided
+var DefaultSchedulerConfig = `
+partitions:
+  - name: default
+    placementrules:
+      - name: tag
+        value: namespace
+        create: true
+    queues:
+      - name: root
```

Comment:
We are missing the following line from the code:
```
          submitacl: '*'
```

Reply:
Updated.